jmx/src/test/resources/logging.properties

```diff
@@ -0,0 +1,57 @@
+#
+# JBoss, Home of Professional Open Source.
+# Copyright 2016, Red Hat, Inc., and individual contributors
+# as indicated by the @author tags. See the copyright.txt file in the
+# distribution for a full listing of individual contributors.
+#
+# This is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; either version 2.1 of
+# the License, or (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this software; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+#
+
+# Additional logger names to configure (root logger is always configured)
+loggers=javax.security.sasl,org.xnio.ssl,org.xnio,org.jboss.remoting,org.jboss.remoting.endpoint
+
+# Root logger configuration
+#logger.level=${test.level:INFO}
+logger.level=INFO
+logger.handlers=CONSOLE
+
+# Configure javax.security.sasl to be less verbose by default
+logger.javax.security.sasl.level=INFO
+
+# Configure org.xnio.ssl to be less verbose by default
+logger.org.xnio.ssl.level=INFO
+
+# XNIO is just over the top with trace logging
+logger.org.xnio.level=DEBUG
+
+# Reduce noise here
+logger.org.jboss.remoting.level=INFO
+
+# Debug endpoint
+logger.org.jboss.remoting.endpoint.level=TRACE
```

Comment:
Please remove this file from PR.
or move logging of all loggers to INFO or higher.